tests/dsl/dtd/dtd_test_cuda_task_insert.c

```diff
@@ -758,67 +754,67 @@ int test_cuda_multiple_devices(int world, int myrank, parsec_context_t *parsec_c
-                                              sizeof(int), PARSEC_VALUE | PARSEC_AFFINITY,
-                                              PARSEC_DTD_ARG_END);
+                    PASSED_BY_REF, PARSEC_INPUT | TILE_FULL,
+                    PASSED_BY_REF, PARSEC_INOUT | TILE_FULL | PARSEC_PUSHOUT,
+                    sizeof(int), PARSEC_VALUE,
+                    sizeof(int), PARSEC_VALUE | PARSEC_AFFINITY,
+                    PARSEC_DTD_ARG_END);
     parsec_dtd_task_class_add_chore(tp, cudacpy_tc, PARSEC_DEV_CUDA, cuda_cpy_task_fn);
 
     for( int rank = 0; rank < world; ++rank ) {
         int prio = 1;
 
         for(int i = 0; i < mt; i+=3) {
-            parsec_data_key_t key_0 = A->data_key(A, i*nt+myrank, 0);
+            parsec_data_key_t key_0 = A->data_key(A, i, myrank);
             uint32_t rank_of_key_0 = A->rank_of_key(A, key_0);
             printf("[dtd_test_cuda][multiple_devices] key_0 = %lu, rank_of_key_0 = %d\n", key_0, rank_of_key_0);
             parsec_dtd_tile_t *tile_0 = parsec_dtd_tile_of(A, key_0);
 
             parsec_advise_data_on_device(
-                                         tile_0->data_copy->original, cuda_dev_index[0],
-                                         PARSEC_DEV_DATA_ADVICE_PREFERRED_DEVICE);
+                    tile_0->data_copy->original, cuda_dev_index[0],
+                    PARSEC_DEV_DATA_ADVICE_PREFERRED_DEVICE);
 
-            parsec_data_key_t key_1 = A->data_key(A, (i + 1)*nt+myrank, 0);
+            parsec_data_key_t key_1 = A->data_key(A, i + 1, myrank);
             uint32_t rank_of_key_1 = A->rank_of_key(A, key_1);
             printf("[dtd_test_cuda][multiple_devices] key_1 = %lu, rank_of_key_1 = %d\n", key_1, rank_of_key_1);
             parsec_dtd_tile_t *tile_1 = parsec_dtd_tile_of(A, key_1);
 
             parsec_advise_data_on_device(
-                                         tile_1->data_copy->original, cuda_dev_index[1],
-                                         PARSEC_DEV_DATA_ADVICE_PREFERRED_DEVICE);
+                    tile_1->data_copy->original, cuda_dev_index[1],
+                    PARSEC_DEV_DATA_ADVICE_PREFERRED_DEVICE);
 
-            parsec_data_key_t key_2 = A->data_key(A, (i + 2)*nt+myrank, 0);
+            parsec_data_key_t key_2 = A->data_key(A, i + 2, myrank);
             uint32_t rank_of_key_2 = A->rank_of_key(A, key_2);
             printf("[dtd_test_cuda][multiple_devices] key_2 = %lu, rank_of_key_2 = %d\n", key_2, rank_of_key_2);
             parsec_dtd_tile_t *tile_2 = parsec_dtd_tile_of(A, key_2);
 
             parsec_advise_data_on_device(
-                                         tile_2->data_copy->original, cuda_dev_index[0],
-                                         PARSEC_DEV_DATA_ADVICE_PREFERRED_DEVICE);
+                    tile_2->data_copy->original, cuda_dev_index[0],
+                    PARSEC_DEV_DATA_ADVICE_PREFERRED_DEVICE);
 
             parsec_dtd_insert_task(
```

Comment:
This PR looks huge, but a lot of these changes are indentation. It is really necessary ?